hudi-timeline-service/src/main/java/org/apache/hudi/timeline/service/TimelineService.java

```diff
@@ -263,10 +265,12 @@ private int startServiceOnPort(int port) throws IOException {
   }
 
   public int startService() throws IOException {
-    final Server server = timelineServerConf.numThreads == DEFAULT_NUM_THREADS ? new Server() : new Server(new QueuedThreadPool(timelineServerConf.numThreads));
-
+    final Server server = timelineServerConf.numThreads == DEFAULT_NUM_THREADS ? new JettyServer(new JavalinConfig()).server() :
```

Comment:
@rahil-c Is the change to `new JettyServer(new JavalinConfig()).server()` due to Javalin upgrade?  I only see minor version upgrade for Javalin and am wondering if this is really needed.

Reply:
@yihua I believe in the past before the recent jetty upgrade to hudi that it was the following https://github.com/apache/hudi/pull/6844/files

```
final Server server = timelineServerConf.numThreads == DEFAULT_NUM_THREADS ? JettyServerUtil.defaultServer()
```
It seems we now opted for `new Server()` and while this might be fine I thought it might be more appropriate to use the closet method to what hudi was using before with `JettyServerUtil.defaultServer` which is creating ` new JettyServer` as the former has been depcreated.

The code of both of these is nearly the same from what i saw 

https://github.com/javalin/javalin/blob/javalin-2.8.0/src/main/java/io/javalin/core/util/JettyServerUtil.kt#L41


https://github.com/javalin/javalin/blob/javalin-parent-4.6.7/javalin/src/main/java/io/javalin/jetty/JettyServer.kt#L33